src/main/resources/com/google/api/codegen/java/settings.snip

```diff
@@ -79,6 +79,12 @@
 
   private static final String DEFAULT_GAPIC_NAME = "gapic";
   private static final String DEFAULT_GAPIC_VERSION = "";
+  private static final String ARTIFACT_ID = "{@xsettingsClass.artifactId}";
+  private static final String GROUP_ID = "com.google.cloud";
```

Comment:
Right this is hard-coded and won't work for non-cloud APIs. But right now there is no way for toolkit to know the maven group ID. We probably need to add it as a field in the API gapic config yaml.